sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/policy/AzureMonitorRedirectCustomPolicy.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter.policy;
```

Comment:
This should be in the root package `com.azure.monitor.opentelemetry.exporter`

Reply:
My initial thought was to have all custom policies related to exporter in a separate package. Any particular reason to have this in root package?

Reply:
If you move it to `com.azure.monitor.opentelemetry.exporter` it can probably be package-protected so that it's not exposed as a public class